### ISSUE TITLE
Suggestion to broaden XDV.1.2 to include everything under S:R346T branch.

### ISSUE THREAD

**Issue by xz-keg** [cov-lineages/pango-designation#2789]
The only XDV.1+S:R346T seqs were the N:I292T branch when XDV.1.2 was designated. 
However as more seqs are submitted there are many more branches now. Suggest to broaden XDV.1.2 accordingly. @AngieHinrichs @corneliusroemer 

https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice2_genome_test_5dd41_689d60.json?label=id:node_5496004
![image](https://github.com/user-attachments/assets/e2f0a013-1ce9-4ed1-98d0-54ee0dfb9f67)

**Comment by FedeGueli** [comment]
> The only XDV.1+S:R346T seqs were the N:I292T branch when XDV.1.2 was designated. However there are many more branches now. Suggest to broaden XDV.1.2 accordingly. @AngieHinrichs @corneliusroemer
> 
> https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice2_genome_test_5dd41_689d60.json?label=id:node_5496004 ![image](https://private-user-images.githubusercontent.com/10445944/371497833-e2f0a013-1ce9-4ed1-98d0-54ee0dfb9f67.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc0NDE3NTAsIm5iZiI6MTcyNzQ0MTQ1MCwicGF0aCI6Ii8xMDQ0NTk0NC8zNzE0OTc4MzMtZTJmMGEwMTMtMWNlOS00ZWQxLTk4ZDAtNTRlZTBkZmI5ZjY3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI3VDEyNTA1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJmN2VlZGQ4NTI2YmRjMWNkMjMyNzQ5MTQ5N2E4NzNlMzUyMzhiYWNiZmYxNDRjNjZiOTY4ODE5Nzc1MzE5YTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.D0hZVISspTQTwYBbcqTwFojMYVXjQqGZ56A0XnjlV08)

They doesnt seem related if not the orf1a:L3116F carrying ones.

**Comment by FedeGueli** [comment]
Or you think they are all related being mostly by Shanxi?

**Comment by aviczhl2** [comment]
> They doesnt seem related if not the orf1a:L3116F carrying ones.

Like JN.1.16.1, they shall be assigned a name despite potential separate occurence.

**Comment by FedeGueli** [comment]
> > They doesnt seem related if not the orf1a:L3116F carrying ones.
> 
> Like JN.1.16.1, they shall be assigned a name despite potential separate occurence.

Yeah but already assigned, it is very rare that a lineage has been broadened or tightened, classical example AY.41  that was designated too deep leavig out a lineage, sibling of AY.103, that  was deadly in eastern Europe during summer 2021.

**Comment by aviczhl2** [comment]
> > > They doesnt seem related if not the orf1a:L3116F carrying ones.
> > 
> > 
> > Like JN.1.16.1, they shall be assigned a name despite potential separate occurence.
> 
> Yeah but already assigned, it is very rare that a lineage has been broadened or tightened, classical example AY.41 that was designated too deep leavig out a lineage, sibling of AY.103, that was deadly in eastern Europe during summer 2021.

They do not have a name and is under the general XDV.1, while XDV.1.2 now starts from  N:I292T

**Comment by FedeGueli** [comment]
> They do not have a name and is under the general XDV.1, while XDV.1.2 now starts from N:I292T

i mean most of them are on XDV.1.2 with starting with 346T it would become the collector of XDV.1 +346T but likely already hard to think there will be new emergence of 346T directly from the politomy after many months around so likely worth broadening it.

**Comment by DailyCovidCases** [comment]
@xz-keg I think you can close this issue now
